### PR TITLE
Replace legacy console URL with Kyma Dashboard

### PR DESCRIPTION
**Pull request by moelsayed**
**Description**

Changes proposed in this pull request:

- Replace legacy console URL with Kyma Dashboard for newly provisioned instances.
- Replace legacy console URL with Kyma Dashboard for existing instances on update.
- Add feature flag for Kyma Dashboard
- Add feature flag to KEB chart

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/instance_create.go [hunk 1]
```diff
@@ -49,6 +49,8 @@ type ProvisionEndpoint struct {
 	shootProject      string
 	shootDnsProviders gardener.DNSProvidersData
 
+	kubeconfigOrigin string
```
Comment (kwiatekus) [inline components/kyma-environment-broker/internal/broker/instance_create.go:52]: This is representing a kyma dashboard URL ( per landscape ).
Why not calling it this `kymaDashboardBaseURL` instead ? I'm confused with `kubeconfigOrigin`

**Review by kwiatekus — CHANGES_REQUESTED** [review]

**Review by kwiatekus — DISMISSED** [review]

**Comment by piotrmiskiewicz** [comment]
please, create a ticket to remove the feature flag when this solution (kyma dashboard) will be enabled in all environmnets. In such ticket we need to remove the feature flag and the old dashboard url logic

**Comment by piotrmiskiewicz** [comment]
this PR changes dashborad url for new SKRs, but old SKRs will still have the old dashboard url

**Comment by kwiatekus** [comment]
> this PR changes dashborad url for new SKRs, but old SKRs will still have the old dashboard url, is it ok?

We should also migrate the labels of existing instancses during upgrade

**Comment by moelsayed** [comment]
@piotrmiskiewicz @kwiatekus Updated to handle existing SKRs, PTAL.

**Review by szwedm — APPROVED** [review]
/lgtm

**Pull request merged**